dev/dags/simple_dag_async.py

```diff
@@ -24,7 +24,7 @@
 simple_dag_async = DbtDag(
     # dbt/cosmos-specific parameters
     project_config=ProjectConfig(
-        DBT_ROOT_PATH / "original_jaffle_shop",
+        DBT_ROOT_PATH / "jaffle_shop",
```

Comment:
Apart from file deletions by removing the `dbt/original_jaffle_shop` folder in the PR, this is one of the two changes we're introducing in this PR. Adding this comment for helping reviewers.